flight/mcu/lib/teensyShared/telemetry.cpp

```diff
@@ -39,45 +42,21 @@ Telemetry::Telemetry() : rf95(RFM95_CS, RFM95_INT) {
     delay(10);
 
     while (!rf95.init()) {
-        Serial.println("Radio Initialization Failed");
-        while (1)
-            ;
+        while (true) {
+            Serial.println("Radio Initialization Failed");
+        }
     }
     Serial.println("[DEBUG]: Radio Initialized");
 
     // Defaults after init are 434.0MHz, 13dBm, Bw = 125 kHz, Cr = 4/5, Sf =
     // 128chips/symbol, CRC on
 
     if (!rf95.setFrequency(RF95_FREQ)) {
-        Serial.println("[ERROR]: Default setFrequency Failed");
-        while (1)
-            ;
+        while (true) {
+            Serial.println("[ERROR]: Default setFrequency Failed");
+        }
```

Comment:
These should return error codes instead of stall (ik the original code stalled, but it shouldn't have either)

Reply:
We could do something similar to Zig's errorset. We have a header file containing an enum, which has 'every' possible error state. Function which can error return a value of this enum, and the calling code eventually hands this error value to a function which matches on the error value and executes the relevant blink sequence/stall. Does this make sense?

Reply:
Yeah returning an error code is the correct way of handling this